akka-remote/src/main/resources/reference.conf

```diff
@@ -1112,6 +1046,67 @@ akka {
         # Refer to `akka.remote.artery.RemoteInstrument` for more information.
         instruments = ${?akka.remote.artery.advanced.instruments} []
 
+        # Only used when transport is aeron-udp
+        aeron {
+          # Periodically log out all Aeron counters. See https://github.com/real-logic/aeron/wiki/Monitoring-and-Debugging#counters
+          # Only used when transport is aeron-udp.
+          // TODO should we create an aeron-udp section for settings specific to aeron-udp?
```

Comment:
Is there anything else than aeron-udp that we support?

Reply:
Doh, this TODO should be removed. This is that section after initial review from P but we decided to call it aeron rather than aeron-udp